config-templates/redhat/odf-local/changelog.json

```diff
@@ -3,43 +3,43 @@
         "4.13": {
             "status": "supported",
             "latest-revision": {
+                "revision": "6",
+                "release-date": "19 January 2023",
+                "cves": "",
+                "golang": "1.20.11",
+                "ubi": "8.9-1029",
+                "new-features": ["Enable/Disable Noobaa public LB"],
+                "fixes": ["fixed uninstallation issues"]
+            },
+            "previous-revision": {
                 "revision": "5",
                 "release-date": "30 October 2023",
                 "cves": "CVE-2023-44487",
                 "golang": "1.20.7",
                 "ubi": "8.8-1072.1697626218",
                 "new-features": [],
                 "fixes": []
-            },
-            "previous-revision": {
-                "revision": "4",
-                "release-date": "19 October 2023",
-                "cves": "CVE-2023-29491, CVE-2023-4911, CVE-2023-4527, CVE-2023-4806, CVE-2023-4813",
-                "golang": "1.20.7",
-                "ubi": "8.8-1072.1696517598",
-                "new-features": [],
-                "fixes": []
             }
         }
     },
     {
         "4.12": {
             "status": "supported",
             "latest-revision": {
-                "revision": "8",
-                "release-date": "27 November 2023",
-                "cves": "CVE-2007-4559, CVE-2023-4641, CVE-2023-22745",
+                "revision": "9",
+                "release-date": "19 January 2023",
+                "cves": "",
```

Comment:
There are CVEs reported older image of 4.12.  Can you please update all CVEs here?

Reply:
There is no CVE reported for the new UBI image published. As the new UBI is available, I moved to that image